src/core/include/antioch/kinetics_conditions.h

```diff
@@ -109,7 +115,14 @@ namespace Antioch{
   inline
   const StateType & KineticsConditions<StateType,VectorStateType>::T() const
   {
-     return _temperature;
+     return _temperature.T;
+  }
+
+  template <typename StateType, typename VectorStateType>
+  inline
+  const StateType & KineticsConditions<StateType,VectorStateType>::Tvib() const
+  {
+     return _temperature.T;
```

Comment:
Maybe I'm missing something, but it seems like this method is for vibrational temperature and we're returning translational temperature.